packages/management/addons-manager/bundle/config/upstream/addons-manager.yaml

```diff
@@ -20,6 +20,17 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Could we add a comment that these permissions are required for addons-manager to create/update resources in the ProviderRef which are dynamic.

Reply:
question: If we have the following:
```
apiGroups:
- ["*"]
resources:
- ["*"]
verbs:
...
```

Would it be a superset of the rest rules ?